lib/generators/boring/overcommit/pre_commit/rubocop/install/install_generator.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "generators/boring/overcommit/base_generator"
+
+module Boring
+  module Overcommit
+    module PreCommit
+      module Rubocop
+        class InstallGenerator < Boring::Overcommit::BaseGenerator
+          def configure_rubocop
+            unless rubocop_gem_exists?
+              say "\nrubocop gem is not installed. Please install it and run the generator again!", :red
```

Comment:
can we run rubocop install generator instead here?

Reply:
I have updated the generator to use RuboCop install generator in this case @abhaynikam